### ISSUE TITLE
A project already exists not detailed message

### ISSUE THREAD

**Issue by danlg** [arunoda/meteor-up-legacy#630]
On my local box, I get  "A Project Already Exists"

dan@localhost:~/DEV/github2/lgen/app$ mup init
## Meteor Up: Production Quality Meteor Deployments

A Project Already Exists

I remove every meteor .meteor project and project n the server side even the meteor binary.
But still get that error message. Any idea ?

**Comment by sherif84** [comment]
I'm having the same issue , can't find any documentation regarding this .

**Comment by MasterJames** [comment]
The code in mupx/lib/actions.js seems to suggest that error means mup.json or setting.json is found.

```
Actions.init  = function() {
  var destMupJson = path.resolve('mup.json');
  var destSettingsJson = path.resolve('settings.json');

  if(fs.existsSync(destMupJson) || fs.existsSync(destSettingsJson)) {
    console.error('A Project Already Exists'.bold.red);
    process.exit(1);
  }
```

**Comment by sherif84** [comment]
Yup , just figured that out , i  deleted the settings.json file and no longer seeing the issue . Thank @MasterJames

**Comment by satyavh** [comment]
Seriously, what a bad programming. And not even fixed after a year... 
Wasted my life to figure this out... :-(
